aioxmpp/muc/xso.py

```diff
@@ -180,7 +180,7 @@ class ActorBase(xso.XSO):
 
     nick = xso.Attr(
         "nick",
-        type_=xso.String(aioxmpp.stringprep.resourceprep),
+        type_=xso.String(precis.Nickname.enforce),
```

Comment:
Fixed in branch (this API was changed with a release yesterday and the top level constants were removed).